crates/topos-node/src/lib.rs

```diff
@@ -0,0 +1,221 @@
+//! Temporary lib exposition for backward topos CLI compatibility
```

Comment:
If this is temporary, what is the long term plan?

Reply:
the long term plan is to have the `topos cli` on one side and the `topos-node` binary that represent the whole runtime of a node